src/post.ts

```diff
@@ -0,0 +1,16 @@
+import * as core from '@actions/core';
+import { Docker } from './model';
+
+export async function run() {
+  try {
+    const runnerTemporaryPath = process.env.RUNNER_TEMP;
+    const githubAction = process.env.GITHUB_ACTION;
+
+    await Docker.cleanup({
+      runnerTemporaryPath,
+      githubAction,
+    });
```

Comment:
I was thinking perhaps we could call this method something with `ensure...` in the name. What do you think?

Reply:
I think `Docker.removeContainer` would be quite explanatory. WDYT?

Reply:
It makes it look like removal always happens in this bit, which isn't the case. I think ensure makes sense here.

Reply:
Addressed.